week2/homework/src/index.js

```diff
@@ -1,3 +1,62 @@
 'use strict';
 
 // TODO: Write the homework code in this file
+
+const program = require('commander');
+const functions = require('./functions');
+
+const list = functions.list;
+const add = functions.add;
+const remove = functions.remove;
+const reset = functions.reset;
+const help = functions.help;
+const update = functions.update;
+
+if (!process.argv[2]) {
+  help();
+  // console.log(process.argv[2]);
+} else {
+  program
+    .command('help')
+    .description('to get help')
+    .action(() => {
+      help();
+    });
+
+  program
+    .command('list')
+    .description('to list to-dos')
+    .action(() => {
+      list();
+    });
+
+  program
+    .command('add <newToDo>')
+    .description('to add a new to-do')
+    .action(newToDo => {
+      add(newToDo);
+    });
+
+  program
+    .command('remove <toDo>')
+    .description('to remove an existing to-do')
+    .action(toDo => {
```

Comment:
Not a big error, but I would suggest to call it "toDoNumber", same variable name used for the update function.
Naming consistency makes it more readable and neat.